docs/contents/community/faq.md

```diff
@@ -12,11 +12,46 @@
 
     We appreciate input and advice concerning features that are currently covered by the service broker only, for example bind/unbind or cataloging and looking forward to learn from the community how they are using service broker so that we can take this into account.
 
+## Cluster API
+
+!!! question "Question"
+    Does ACK replace [cluster api](https://github.com/kubernetes-sigs/cluster-api)?
+
+!!! quote "Answer"
+    Cluster API does a lot of really cool things and is designed to be a generic way to create Kubernetes clusters that run anywhere.
```

Comment:
I would start this answer with a specific: "No, the ACK service controller for EKS does not replace Kubernetes Cluster API."

and then continue with another paragraph about "Cluster API does a lot of really cool things..."

Reply:
Yep, what Jay said. We should strive to always clearly say Yes or No in the first sentence (if it's a binary question).